modules/Registration/components/student/StudentRegistrationForm.helpers.ts

```diff
@@ -3,6 +3,8 @@ import { z } from "zod";
 
 import { TStudentRegistrationFormData } from "./StudentRegistrationForm.types";
 
+const bangladeshPhoneRegex = /^(?:\+8801|01)[3-9]\d{8}$/;
```

Comment:
This is likely too stringent. I would suggest bring it up in the next standup to discuss the requirements

Reply:
Yes bhaiya, I was planning to discuss this requirement in the next standup.